support/nodeJSVersion.js

```diff
@@ -0,0 +1,8 @@
+
+module.exports = function nodeJSVersion(arg) {
```

Comment:
can you elaborate ?
I found that detecting the  nodeJSVersion was needed because in version 10 and 12 of NodeJS , the length is not increased although the function length is configurable; I needed a way to detect this 2 special case in the unit test.

Reply:
I'm skeptical - in my experimentation, it works fine in node 10:
```js
function f() {}
Object.getOwnPropertyDescriptor(f, 'length').configurable; // true
Object.defineProperty(f, 'length', { value: 42 });
f.length; // 42
```

but then we should be feature-detecting this failure. we shouldn't be hardcoding specific versions of things, especially since any issue in specific node versions also matches issues in some Chrome versions.

Reply:
@ljharb , yes fct.length is configurable in node10  onward but the util.callbackify  do not take advantage of this and keep the fct.length unchanged. the behavior has been fixed by the nodejs team from node12 onwards.

> we shouldn't be hardcoding specific versions of things,
I have remove the code that check against a specific version of node in `util.js` .

Let me know if there is anything else I can do .

Reply:
Then we should be feature-detecting when the length is configurable but when util.callbackify fails to change the length. The node version should never be checked in runtime code.

Reply:
this still seems an issue, since this file shouldn't exist